website_sale_checkout_skip_payment/controllers/main.py

```diff
@@ -36,10 +30,6 @@ def shop_payment_confirmation(self, **post):
             return request.render(
                 "website_sale_checkout_skip_payment.confirmation_order_error"
             )
-        # This could not finish (e.g.: sale_financial_risk exceeded)
-        order.action_confirm()
```

Comment:
This is a big behavior change. Why removing it?

Reply:
Ok, I was restoring the original logic and missed that step. It's correct now, thanks for the heads up